MANIFEST.in

```diff
@@ -5,7 +5,7 @@ include dash_core_components/dash_core_components.dev.js.map
 include dash_core_components/metadata.json
 include dash_core_components/rc-slider@6.1.2.css
 include dash_core_components/react-select@1.0.0-rc.3.min.css
-include dash_core_components/plotly-1.45.0.min.js
+include dash_core_components/plotly-*.min.js
```

Comment:
Using wildcard like here https://github.com/plotly/dash-renderer/blob/master/MANIFEST.in